tests/helper/helper_http.go

```diff
@@ -21,6 +21,8 @@ func HttpWaitFor(url string, match string, maxRetry int, interval int) {
 	for i := 0; i < maxRetry; i++ {
 		fmt.Fprintf(GinkgoWriter, "try %d of %d\n", i, maxRetry)
 
+		// #nosec
+		// gosec:G107 -> This is safe since it's just used for testing.
```

Comment:
```
G107: Url provided to HTTP request as taint input
```
@JAORMX I think the the comment you added is not what it actually for. It is passing through the ```sec``` check because the ```http.Get(url)``` doesn't use url as an argument of a sub process call. 

So the comment does not make sense here.

Reply:
Right; my intention with the comment was to suggest that we can safely ignore the warning because the call is just used for the test suite.

Reply:
Understood, Please remove those two lines of comment to avoid the confusion
Thnaks

Reply:
@amitkrout // #nosec is needed because otherwise, the gosec tool that I'm introducing as part of this PR will generate a warning. The line after it is explaining why it's safe to ignore that security warning.

Reply:
Hmm, Good point.
Lets keep it then